util/kube/portforwarder.go

```diff
@@ -40,21 +40,29 @@ func PortForward(podSelector string, targetPort int, namespace string, overrides
 		return -1, err
 	}
 
-	pods, err := clientSet.CoreV1().Pods(namespace).List(context.Background(), v1.ListOptions{
-		LabelSelector: podSelector,
-	})
-	if err != nil {
-		return -1, err
+	var pod *corev1.Pod
+
+	for _, podSelector := range podSelectors {
+		pods, err := clientSet.CoreV1().Pods(namespace).List(context.Background(), v1.ListOptions{
+			LabelSelector: podSelector,
+		})
+		if err != nil {
+			return -1, err
+		}
+
+		if len(pods.Items) > 0 {
+			pod = &pods.Items[0]
```

Comment:
Do we want to break out of this for loop once we found a pod?

Reply:
thank you! fixed